package.json

```diff
@@ -114,6 +114,7 @@
     "express": "^4.17.1",
     "lodash.throttle": "^4.1.1",
     "marinate": "^0.1.5",
+    "postcss-root-to-json": "0.0.0",
```

Comment:
where is this used?

Reply:
Not sure, it was something added in automatically. https://developer.aliyun.com/mirror/npm/package/postcss-root-to-json